GETTING_STARTED.md

```diff
@@ -0,0 +1,62 @@
+<br> <i> Languages: </i> JavaScript
+
+### The Workflow
+
+The user launches <b>Ishiiruka</b> and selects an .iso of Melee to emulate. Upon lauching the emulation of Melee, Ishiiruka injects the modifications made by <b> Slippi SSBM ASM </b>. As the user interacts with the game, information is exchanged between Ishiiruka and the Slippi SSBM ASM code. As a user begins an online match, Ishiiruka starts a log of in-game state reported by Slippi SSBM ASM. This log is ultimately formatted and written to a .slp file, according to the [Slippi replay file spec](https://github.com/project-slippi/slippi-wiki). These files are then viewable from the <b> Slippi Desktop App </b>. From the desktop app, a user may choose to launch the replay, in which case Ishiiruka is launched in a replay mode. A user may also choose to view details or statistics about a previously played match, in order to do so, the desktop app leverages the <b> Slippi-JS </b> library.  
+
+
+## Commonly Asked Questions
+> "Where is the rollback code located?"  
+
+Rollback is accomplished by work done between the [Slippi SSBM Asm](https://github.com/project-slippi/slippi-ssbm-asm/search?p=1&q=rollback&unscoped_q=rollback) code and the [Ishiiruka](https://github.com/project-slippi/Ishiiruka/search?q=rollback&unscoped_q=rollback) code.
+
+> "How is data moved between the game (assembly) and the emulator?"
+
+Via EXI communication. An example of such is demonstrated in [this video](https://github.com/project-slippi/Ishiiruka/search?q=rollback&unscoped_q=rollback) by Fizzi.
+
+> "Where is the matchmaking code?"
+
+The source code for the matchmaking server is in a private repository, and is not currently planned to be open sourced.
+
+## Guides & Resources
+
+### Melee Modding and Assembly
+* [Fizzi Teaches Basic Assembly and Melee Modding](https://www.youtube.com/watch?v=NOq49h0tkBI) (video) - Recorded by the creator of Slippi, this video teaches some basic assembly and demonstrates EXI communication.
+* [SSBM Data Sheet](https://docs.google.com/spreadsheets/d/1JX2w-r2fuvWuNgGb6D3Cs4wHQKLFegZe2jhbBuIhCG8/preview#gid=12) - A Google Sheet containing information about Melee's memory addresses.
+* [Assembler Tutorial](http://wiibrew.org/wiki/Assembler_Tutorial) - A wii modding specific guide to PPC, and its instructions. 
+* [Intro to Wii Game Modding by InternetExplorer](https://www.youtube.com/watch?v=IOyQhK2OCs0&list=PL6GfYYW69Pa2L8ZuT5lGrJoC8wOWvbIQv) (video) - Recorded by Dan Salvato - A playlist of videos related to modding Wii games. An excellent resource for debugging with Dolphin.
+
+## Tools
+
+### Melee Modding and Assembly
+* [HxD](https://mh-nexus.de/en/hxd/) - A free Hex editor. Hex editors in general are useful for looking through memory dumps. 
+* [SpeedCrunch](https://speedcrunch.org/) - A calculator for programmers. It allows for quick conversions between hex, binary, octal, and decimal. 
```

Comment:
Windows' standard calculator also supports this btw.

Reply:
> 
> 
> Windows' standard calculator also supports this btw.

I guess more notably, you can perform operations on numbers of different bases. I find it pretty useful for adding decimal offsets, to memory addresses - which are in hex. It also has configurable constants, so it's been useful to have pre-defined memory locations that I can reference without having to copy and paste. It's also available on macOS and Linux.